pkg/innerring/internal/blockchain/blockchain.go

```diff
@@ -346,6 +379,15 @@ func New(cfg Config) (res *Blockchain, err error) {
 	cfgBaseApp.P2P.AttemptConnPeers = int(cfg.P2P.AttemptConnPeers)
 	cfgBaseApp.P2P.MaxPeers = int(cfg.P2P.MaxPeers)
 
+	cfgBaseApp.RPC.Enabled = true
```

Comment:
it's not always enabled

Reply:
then why `Blockchain` declares it optional?

Reply:
It's absolutely always has to be enabled. It's just that it may have no addresses to listen (neither http, nor https), but the service itself has to be enabled.

Reply:
what the service does w/o listening?

Reply:
Provides local client that is used by IR.